cloudnetpy/instruments/mira.py

```diff
@@ -60,9 +60,12 @@ def mira2nc(
     }
 
     with TemporaryDirectory() as temp_dir:
-        if os.path.isdir(raw_mira):
+        if type(raw_mira) is list or os.path.isdir(raw_mira):
```

Comment:
As indicated by Pylint, you should use `isinstance` unless there's a very good reason.